tests/actor/power_distributing/test_power_distributing.py

```diff
@@ -1167,6 +1197,8 @@ async def test_power_distributor_one_user_adjust_power_success(
         assert result.excess_power.isclose(Power.zero(), abs_tol=1e-9)
         assert result.request == request
 
+        await mockgrid.cleanup()
+
     async def test_not_all_batteries_are_working(self, mocker: MockerFixture) -> None:
```

Comment:
I guess this last test in the file could also use a cleanup call.  It doesn't seem to have one now.

Reply:
yes, it should have it. Good catch!

Reply:
Added